example/viam_example_app/lib/screens/stream.dart

```diff
@@ -73,35 +65,6 @@ class _StreamScreenState extends State<StreamScreen> {
     return uiImage;
   }
 
-  Future<void> _startStream() async {
```

Comment:
why is this all being deleted?

Reply:
because I replaced it with the widget which has this all in a reusable class